aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -596,6 +603,64 @@ class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknownHtmlT
         return savedState
     }
 
+    private fun cacheFilenameKey(varName: String): String {
+        return "CACHEFILENAMEKEY_$varName"
+    }
+
+    private fun writeTempInstance(varName: String, obj: Any?, bundle: Bundle) {
+        try {
+            with(File.createTempFile(varName, ".inst", context.getCacheDir())) {
+                deleteOnExit() // just make sure if we miss deleting this cache file the VM will eventually do it
+
+                FileOutputStream(this).use { output ->
+                    ObjectOutputStream(output).use { objectOutput ->
+                        objectOutput.writeObject(obj)
+
+                        // keep the filename in the bundle to use it to read the object back
+                        bundle.putString(cacheFilenameKey(varName), this.path)
+                    }
+                }
+            }
+        } catch (e: IOException) {
+            AppLog.w(AppLog.T.EDITOR, "Error trying to write cache for $varName. Exception: ${e.message}")
```

Comment:
What do you think on logging errors about accessing the file in Crashlytics so we can see how many users have problem with this new method of writing state?
`externalLogger?.logException(e, "Error trying to write cache for $varName.")`

Reply:
When Aztec is integrated in `wp-android` the external logger is Crashlytics. The exception is wrapped in a special Aztec Exception and easily searchable in the web interface (I hope).

Reply:
Makes sense @daniloercoli , will work on this on a separate PR since this one got merged already. 👍